dominos/src/main/java/me/fontys/semester4/dominos/configuration/data/store/StoreImportBootstrap.java

```diff
@@ -0,0 +1,28 @@
+package me.fontys.semester4.dominos.configuration.data.store;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+import java.io.IOException;
+
+@Configuration
+public class StoreImportBootstrap {
+
+    @Value("${import.order.wildcard}")
```

Comment:
Copy pasta? 😛
Probably want to change that to `import.store.wildcard`. Wildcard is likely not even necessary here (but can be included, just for the sake), as you only have one store file currently.